data/summary.go

```diff
@@ -0,0 +1,19 @@
+package data
+
+import (
+    "time"
+    "go.mongodb.org/mongo-driver/bson/primitive"
+)
+
+type Summary struct {
+    ID                 primitive.ObjectID  `json:"-" bson:"_id,omitempty"`
+	UserID             string              `json:"UserID" bson:"_userId"`
```

Comment:
Nit: consider renaming this field `userId` for consistency with existing APIs.
To be fair, we're already inconsistent with `userid`, `userId`, and `_userId` - but I think `userID` would be new.